final part 1

NMandel527 opened the pull request.

andrewoid commented:
How do you have merge conflicts?

andrewoid commented on build.gradle:
```diff
@@ -1,9 +1,60 @@
 plugins {
-    id 'java'
+    id 'java-library'
+    id "com.vanniktech.maven.publish" version "0.28.0"
```
You were never told to add any of this

NMandel527 replied:
I was looking at the code in the src file that was sent with the information about apis. Should I delete the code that I added from there?

andrewoid replied:
You should. It will screw up everything else.

NMandel527 replied:
Ok will do

andrewoid commented on src/main/java/mandel/museum/ArtObjects.java:
```diff
@@ -0,0 +1,10 @@
+package mandel.museum;
+
+public class ArtObjects
```
Singular, not plural

andrewoid commented on src/main/java/mandel/museum/RijksService.java:
```diff
@@ -0,0 +1,28 @@
+public interface RijksService
+{
+    @GET("/api/en/collection")
+    Single<Art> artP(
```
Change these methods names so they mean something

andrewoid commented on src/test/java/mandel/museum/RijksServiceFactoryTest.java:
```diff
@@ -0,0 +1,71 @@
+        ).blockingGet();
+
+        //then
+        assertNotNull(artP.artObjects[0]);
```
Test what is inside the artobject

NMandel527 commented:
> How do you have merge conflicts?
It's not letting me reply directly, but I was just looking at the merge conflict. I'm not sure what happened. It was fine and then I updated it by adding a readme and this came up.

andrewoid commented:
> > How do you have merge conflicts?
> It's not letting me reply directly, but I was just looking at the merge conflict. I'm not sure what happened. It was fine and then I updated it by adding a readme and this came up.
> 

How did you add a Readme?

NMandel527 commented:
> > > How do you have merge conflicts?
> > > It's not letting me reply directly, but I was just looking at the merge conflict. I'm not sure what happened. It was fine and then I updated it by adding a readme and this came up.
> 
> How did you add a Readme?

I added a new file called README.md and then I added information to it. Then I did git add . git commit -m "added readme" git push origin api

andrewoid commented:
> > > > It's not letting me reply directly, but I was just looking at the merge conflict. I'm not sure what happened. It was fine and then I updated it by adding a readme and this came up.

It doesnt tell me what the conflict is

NMandel527 commented:
> > > > > It's not letting me reply directly, but I was just looking at the merge conflict. I'm not sure what happened. It was fine and then I updated it by adding a readme and this came up.
> 
> It doesnt tell me what the conflict is

This is all it's telling me. Is there a way for me to find out more?
![Screenshot 2024-05-15 225142](https://github.com/NMandel527/Mandel-Museum-2024/assets/140018768/e4f48343-bfba-4d94-bfc5-487ef4abd927)

andrewoid commented:
> > > > > > It's not letting me reply directly, but I was just looking at the merge conflict. I'm not sure what happened. It was fine and then I updated it by adding a readme and this came up.
> > 
> > It doesnt tell me what the conflict is
> 
> This is all it's telling me. Is there a way for me to find out more?
> ![Screenshot 2024-05-15 225142](https://github.com/NMandel527/Mandel-Museum-2024/assets/140018768/e4f48343-bfba-4d94-bfc5-487ef4abd927)
> 

You and I will need to meet tomorrow and resolve this.

NMandel527 commented:
> > > > > > > It's not letting me reply directly, but I was just looking at the merge conflict. I'm not sure what happened. It was fine and then I updated it by adding a readme and this came up.
> > > 
> > > 
> > > It doesnt tell me what the conflict is
> > 
> > 
> > This is all it's telling me. Is there a way for me to find out more?
> > ![Screenshot 2024-05-15 225142](https://private-user-images.githubusercontent.com/140018768/331031182-e4f48343-bfba-4d94-bfc5-487ef4abd927.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU4Mjg0MjYsIm5iZiI6MTcxNTgyODEyNiwicGF0aCI6Ii8xNDAwMTg3NjgvMzMxMDMxMTgyLWU0ZjQ4MzQzLWJmYmEtNGQ5NC1iZmM1LTQ4N2VmNGFiZDkyNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTE2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxNlQwMjU1MjZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kMzVjMWUxZmZlNjdjZDYxODFiZDAzMTg1ODM2MGY1ODE4NjAzZWVlZDY0YjUwODdhNjYxYmFkNDQ3Njk0NWNhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.jcjSBcHm3CW0XCmL77aZc8gLUVKjNVdu8LbjYxGOgrs)
> 
> You and I will need to meet tomorrow and resolve this.

Ok thank you. What time works best?

andrewoid commented:
Email me in the morning.

NMandel527 commented:
> Email me in the morning.

Ok thank you so much
